sailor/utils/config.py

```diff
@@ -47,6 +47,8 @@ def get(*keys):
         if SailorConfig.config is None:
             SailorConfig.load()
         res = getattr(SailorConfig.config, keys[0])
+        if res is None:
+            raise RuntimeError(f"Top-level property '{keys[0]}' not found in config.")
```

Comment:
I'd rather have this as `You have not configured credentials for '{keys[0]}', can not retrieve data.` or some such